server_development/topics/auth-spi.adoc

```diff
@@ -374,8 +374,8 @@ getReferenceCategory() is just a category the Authenticator belongs to.
 ==== Adding Authenticator Form
 
 Keycloak comes with a Freemarker <<_themes,theme and template engine>>.
-The createForm() method you called within authenticate() of your Authenticator class, builds an HTML page from a file within your login theme: secret-question.ftl.
-This file should be placed in the login theme with all the other .ftl files you see for login. 
+The createForm() method you called within authenticate() of your Authenticator class, builds an HTML page from a file within your login theme: `secret-question.ftl`.
+This file should be made available by creating a custom <<_theme_resource,Theme Resource Provider>>.
```

Comment:
@stianst The last sentence could be possibly changed, that now you don't necessarily Theme Resource Provider anymore? WDYT?